Gemfile

```diff
@@ -22,7 +26,7 @@ gem "omniauth-microsoft-office365"
 gem 'omniauth-rails_csrf_protection'
 gem "view_component"
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 7.0.8.7"
+gem "rails", "~> 7.2.2.1"
```

Comment:
```suggestion
gem "rails", "~> 7.2"
```

Make the version flexible

Reply:
I did it 7.2.2.1 to answer to [Possible Content Security Policy bypass in Action Dispatch](https://github.com/advisories/GHSA-vfm5-rmrh-j26v) 
kindly look at it and tell me if you'd still want it to be 7.2

Reply:
https://guides.rubygems.org/patterns/#pessimistic-version-constraint You can update minor versions in Gemfile.lock

Reply:
okay for now let me just change it to Rails 7.2 and see to it